app/src/main/java/com/bunbeauty/papakarlo/common/ui/element/card/NavigationIconCardWithDivider.kt

```diff
@@ -0,0 +1,95 @@
+package com.bunbeauty.papakarlo.common.ui.element.card
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.bunbeauty.papakarlo.R
+import com.bunbeauty.papakarlo.common.ui.element.FoodDeliveryDivider
+import com.bunbeauty.papakarlo.common.ui.element.OverflowingText
+import com.bunbeauty.papakarlo.common.ui.icon16
+import com.bunbeauty.papakarlo.common.ui.icon24
+import com.bunbeauty.papakarlo.common.ui.theme.FoodDeliveryTheme
+
+@Composable
+fun NavigationIconCardWithDivider(
+    modifier: Modifier = Modifier,
+    @DrawableRes iconId: Int,
+    @StringRes iconDescriptionStringId: Int? = null,
```

Comment:
if `iconId` is required then let's make `iconDescriptionStringId` required too.
Or don't pass it on at all.